src/makeSortable.ts

```diff
@@ -50,48 +79,54 @@ export function makeSortable<T>(
                                 dom.select(defaults.handle).events('touchstart')
                             )
                             .map(augmentEvent)
-                    ) as Stream<MouseEvent>;
+                    );
+                    const up$ = xs.merge(
+                        xs.fromObservable(
+                            dom.select('body').events('mouseleave')
+                        ),
+                        xs.fromObservable(dom.select('body').events('mouseup')),
+                        xs.fromObservable(
+                            dom.select(defaults.handle).events('touchend')
+                        )
+                    );
+
+                    const move$ = xs.merge(
+                        xs.fromObservable(
+                            dom.select('body').events('mousemove')
+                        ),
+                        xs
+                            .fromObservable(
+                                dom.select(defaults.handle).events('touchmove')
+                            )
+                            .map(augmentEvent)
+                    );
 
-                    const mouseup$: Stream<MouseEvent> = mousedown$
-                        .mapTo(
+                    const mousedown$: Stream<MouseEvent> = down$
+                        .map(ev =>
                             xs
-                                .merge(
-                                    xs.fromObservable(
-                                        dom.select('body').events('mouseleave')
-                                    ),
-                                    xs.fromObservable(
-                                        dom.select('body').events('mouseup')
-                                    ),
-                                    xs.fromObservable(
-                                        dom
-                                            .select(defaults.handle)
-                                            .events('touchend')
-                                    )
-                                )
-                                .take(1)
+                                .of(ev)
+                                .compose(delay(options.selectionDelay))
```

Comment:
TS didn't complain because an 'options' variable exists in the scope.  'defaults' is the normalized version of 'options' which contains the defaults.